src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraDistributionUIConfig.java

```diff
@@ -30,18 +30,32 @@
 
 import com.synopsys.integration.alert.channel.jira.JiraChannel;
 import com.synopsys.integration.alert.common.descriptor.DescriptorMap;
+import com.synopsys.integration.alert.common.descriptor.config.field.CheckboxConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.field.TextInputConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.ui.ChannelDistributionUIConfig;
 
 @Component
 public class JiraDistributionUIConfig extends ChannelDistributionUIConfig {
+
+    public static final String LABEL_ADD_COMMENTS = "Add Comments";
+    public static final String LABEL_ISSUE_CREATOR = "Issue Creator";
+    public static final String LABEL_JIRA_PROJECT = "Jira Project";
+
+    public static final String DESCRIPTION_ADD_COMMENTS = "If true, this will add comments to the Jira ticket with data describing the latest change to the ticket due to the job processing.";
+    public static final String DESCRIPTION_ISSUE_CREATOR = "The user name of the Jira Cloud user to assign to the issue creator field of the Jira ticket.";
```

Comment:
The user name of the Jira Cloud user to assign _as_ the issue creator field of the Jira ticket.